packages/core-sdk/resolvers/protocol-resolvers/resolvers/ethereum/src/query/functions/resolveProtocol.ts

```diff
@@ -41,6 +41,12 @@ export function resolveProtocol(input: Input_resolveProtocol): ProtocolResolver_
     return supportedProtocolsMap.get("cream_v1");
   } else if (input.token.name == "SushiBar") {
     return supportedProtocolsMap.get("sushibar_v1");
+  } else if (input.token.name.includes("Smart Token Relay")) {
+    return supportedProtocolsMap.get("bancor_v2.1");
+  } else if (input.token.name.includes("Smart Token Relay")) {
+    return supportedProtocolsMap.get("bancor_v2");
+  } else if (input.token.name.includes("Liquidity Pool1")) {
+    return supportedProtocolsMap.get("bancor_v1");
```

Comment:
This also seems a pretty generic name and can exist in other protocols which can cause issues.

Reply:
Yes, I think you're right. I'll see if I can find a more distinct way to identify the tokens.